cwd/app.js

```diff
@@ -217,15 +220,26 @@
   function startKiosk(duration) {
     let views = allGlyphs.filter(obj => obj.view).map(obj => obj.view);
     let index = 0;
+    let hashes = ['#glsc', '#lake', '#air'];
+    if (window.location.hash) {
+      if (!hashes.includes(window.location.hash)) {
+        console.error('Invalid hash: ' + window.location.hash);
+        window.location.hash = '';
```

Comment:
Have you confirmed that this is the correct way to turn off the hash entirely?  I am not sure what our ideal business logic here is, but it sounds nice to me to turn off the hash locking if we somehow reach this case.  (I don't really see how this would happen, honestly, if we pull available hashes out of the database)

Reply:
Yes, I tested this and it does turn off hash locking. There may still be a "#" present in the URL, but since it is "empty" the views will not lock at all. This probably isn't necessary, but I added it just in case as a fail safe.

Reply:
Great!